CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 Not yet released; provisionally v1.4.0 (may change).
 
+### Remove Unsupported Hashers
+
+The following hashers have been removed because of security vulnerabilities and
+performance issues:
+
+- `OBJECT_RFC6962_SHA256` - Security vulnerability
+- `CONIKS_SHA256` - Less than ideal performance 
```

Comment:
Add a recommendation that anyone using this ought to migrate to the 512/256 version?